controller/config.go

```diff
@@ -1062,7 +1064,11 @@ func (c Config) SkipConfirmation() bool {
 	if v, ok := c[SkipConfirmation]; ok {
 		return v.(bool)
 	}
-	return DefaultSkipConfirmation
+	if environmentValue := os.Getenv(osenv.JujuSkipConfirmationEnvKey); environmentValue != "" {
```

Comment:
I'm a little unsure about this on second look. In 3.0, it doesn't make too much sense. If the envvar only gives you a way to enable skip confirmation, but the default is to enable skip confirmation, it has no effect. 

However, when will change the default to false in 3.1, this will make much more sense